cmd/opm/migrate/cmd.go

```diff
@@ -51,5 +52,10 @@ parsers that assume that a file contains exactly one valid JSON object.
 		},
 	}
 	cmd.Flags().StringVarP(&output, "output", "o", "json", "Output format (json|yaml)")
+	cmd.Flags().StringVar(&migrate.Level, "migrate-level", "", "Name of the last migration to run (default: none)\n"+migrations.HelpText())
+	_ = cmd.RegisterFlagCompletionFunc("migrate-level", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return []string{migrations.HelpText()}, cobra.ShellCompDirectiveDefault
```

Comment:
What does this do? I understand at a high level that it helps with shell completion when a user is tabbing in their CLI after typing `--migrate <TAB>`.

But what happens with that? Will a user be able to tab through the names of each level?

Reply:
This was a false start.  I'd hoped that it would allow us to provide a list of available migrators as a tab-completion, but it doesn't work except for bash, and I'm generally disenchanted with it.  
I left in, in case someone was going to point out my error, since I exhausted search results. 
I'll kill it, at least for now.